package.json

```diff
@@ -12,22 +12,22 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "15.1.0",
-    "@rollup/plugin-node-resolve": "9.0.0",
-    "@rollup/plugin-typescript": "6.0.0",
+    "@rollup/plugin-commonjs": "21.0.1",
```

Comment:
I bumped on the dependencies, including `obsidian` which now points to the latest commit for now. This is needed to use `setInstructions` on the EditorSuggest from the latest API changes.